e2e-tests/portal/cypress/e2e/journeys/dashboard/dashboard-deals/dashboard-deals-filter-by-multiple-fields-multiple-values.spec.js

```diff
@@ -172,4 +172,78 @@ context('Dashboard Deals filters - filter by multiple fields with multiple value
+    dashboardDeals.visit();
+
+    filters.showHideButton().click();
+    dashboardDeals.filters.panel.form.status.draft.checkbox().click();
+    dashboardDeals.filters.panel.form.status.readyForChecker.checkbox().click();
+    filters.panel.form.applyFiltersButton().click();
+
+    cy.url().should('eq', relative('/dashboard/deals/0'));
+
+    const EXPECTED_DEALS = ALL_DEALS.filter(({ status }) =>
+      status === CONSTANTS.DEALS.DEAL_STATUS.DRAFT || status === CONSTANTS.DEALS.DEAL_STATUS.READY_FOR_APPROVAL);
+
+    dashboardDeals.rows().should('have.length', EXPECTED_DEALS.length);
+  });
+
+  it('renders only deals that have matching fields - AIN, MIA', () => {
+    cy.login(BANK1_MAKER1);
+    dashboardDeals.visit();
+
+    filters.showHideButton().click();
+    dashboardDeals.filters.panel.form.submissionType.MIA.checkbox().click();
+    dashboardDeals.filters.panel.form.submissionType.AIN.checkbox().click();
+    filters.panel.form.applyFiltersButton().click();
+
+    cy.url().should('eq', relative('/dashboard/deals/0'));
+
+    const EXPECTED_DEALS = ALL_DEALS.filter(({ submissionType }) =>
+      submissionType === CONSTANTS.DEALS.SUBMISSION_TYPE.MIA || submissionType === CONSTANTS.DEALS.SUBMISSION_TYPE.AIN);
+
+    dashboardDeals.rows().should('have.length', EXPECTED_DEALS.length);
+  });
+
+  it('renders only deals that have matching fields for all filters', () => {
+    cy.login(BANK1_MAKER1);
+    dashboardDeals.visit();
+
+    filters.showHideButton().click();
+    dashboardDeals.filters.panel.form.status.draft.checkbox().click();
+    dashboardDeals.filters.panel.form.status.readyForChecker.checkbox().click();
+    dashboardDeals.filters.panel.form.status.makerInputRequired.checkbox().click();
+    dashboardDeals.filters.panel.form.status.submitted.checkbox().click();
+    dashboardDeals.filters.panel.form.status.acknowledgedByUKEF.checkbox().click();
+    dashboardDeals.filters.panel.form.status.inProgressByUKEF.checkbox().click();
+    dashboardDeals.filters.panel.form.status.acceptedByUKEFWithConditions.checkbox().click();
+    dashboardDeals.filters.panel.form.status.acceptedByUKEFWithoutConditions.checkbox().click();
+    dashboardDeals.filters.panel.form.status.rejectedByUKEF.checkbox().click();
+    dashboardDeals.filters.panel.form.status.abandoned.checkbox().click();
+    dashboardDeals.filters.panel.form.submissionType.MIA.checkbox().click();
+    dashboardDeals.filters.panel.form.submissionType.AIN.checkbox().click();
+    dashboardDeals.filters.panel.form.submissionType.MIN.checkbox().click();
+    dashboardDeals.filters.panel.form.dealType.gef.checkbox().click();
+    dashboardDeals.filters.panel.form.dealType.bssEwcs.checkbox().click();
+
+    filters.panel.form.applyFiltersButton().click();
+
+    cy.url().should('eq', relative('/dashboard/deals/0'));
+
+    const EXPECTED_DEALS = ALL_DEALS.filter(({ dealType, status, submissionType }) =>
+      (submissionType === CONSTANTS.DEALS.SUBMISSION_TYPE.MIA
```

Comment:
Can we please add a case where filters are added and then removed.

Reply:
done in another test dashboard-facilities-main-container-filter-remove.spec and dashboard-deals-main-container-filter-remove.spec